canonical/canonical.com
# WD-1174 fix identity and paths

## Done

Fix identity and other paths
Cleaning up code by moving all company-culture pages together.

## QA

- Check these links work as usual:
https://canonical-com-922.demos.haus/careers/company-culture/remote-work
https://canonical-com-922.demos.haus/careers/company-culture/progression
https://canonical-com-922.demos.haus/careers/company-culture/sustainability
https://canonical-com-922.demos.haus/careers/company-culture/diversity
https://canonical-com-922.demos.haus/careers/company-culture

- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4134?atlOrigin=eyJpIjoiNzY0ZDlkYjg5ZmIwNGVhN2JhN2VkNWYxY2M2NTcwYWMiLCJwIjoiaiJ9
